src/components/BaseTimer.js

```diff
@@ -63,13 +66,19 @@ class BaseTimer extends Component {
         this.setState({
           currentTime: newTime,
         });
+        // eslint-disable-next-line react/destructuring-assignment
+        const remainingTime = countDown ? newTime : this.state.setTime - newTime;
+        const beepTimes = [3 * ONE_SECOND, 2 * ONE_SECOND, ONE_SECOND];
+        if (beepTimes.includes(remainingTime)) {
+          this.beep('C4', '8n');
```

Comment:
Don't love this.

Initial design calls the timer interval every 10 milliseconds. This leads to there being no distinguishable pause between beeps if I just do remaining time < 3 seconds. 

Can consider calling the timer method less frequently but not sure what impact that will have on timer accuracy.